app.py

```diff
@@ -1,20 +1,129 @@
 from flask import Flask, render_template, request, redirect, url_for
+import config as cf
+from model import Item
+
 import session_items as session
+import requests
+import json
+
 
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
+#load_dotenv()
 
 @app.route('/')
 def index():
-    items = session.get_items()
-    return render_template('index.html', items=items)
-    #return 'Hello World!'
+    board_id = cf.get_trello_board_id()
+    url = f"https://api.trello.com/1/boards/{board_id}/cards"
+    query = cf.get_trello_query()
+    
+    response = requests.request(
+        "GET",
+        url,
+        params=query
+        )
+    
+    
+    cards = json.loads(response.text)
+    items_list = list()
+    for card in cards:
+        status = 'To Do' 
+        if card['idList'] == '5f5a4b008a129438843fcf10':
+            status = 'To Do'
+        else:
+            status = 'Done'
+
+        item = Item(card['id'], status, card['name'])
+        items_list.append(item)
+
+    return render_template('index.html', items=items_list)
 
-@app.route('/add', methods=['POST'])
-def addToDo():
-    title = request.form['title']
-    session.add_item(title)
-    return redirect(url_for('index'))
 
 if __name__ == '__main__':
     app.run()
+
+
+"""
+Add a complete_item route that accepts the ID of an item as a
+parameter and then calls a method to change its status from 'To
+Do' to 'Done'
+curl -i "localhost:5000/api/foo?a=hello&b=world" 
+"""
+@app.route('/complete_item/<idCard>', methods=['PUT'])
+def update_card(idCard):
+
+    add_card_to_done()
+
+    url = "https://api.trello.com/1/cards/{idCard}"
```

Comment:
You need an `f` before the first quotes.

You are currently trying to send a request to `https://api.trello.com/1/cards/{idCard}` rather than e.g. `https://api.trello.com/1/cards/5f3da4395c9bab8cc9a706b2`